packages/twenty-front/src/utils/create-required-context.ts

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+
+export const createRequiredContext = <TContext>(name: string) => {
+  const Context = React.createContext<TContext | undefined>(undefined);
+  Context.displayName = `${name}Provider`;
+
+  const useRequiredContext = (): TContext => {
```

Comment:
Isn't the name of the context already accessible on the object created by the standard createContext ? Seems like we could only create a  useContextOrThrow hook ?

Reply:
It's possible to only create a hook called `useRequiredContext`, but I rather prefer doing this util as this one is avoiding the direct use of `useContext` with a required context.

Reply:
oh ok, just read your comment above! I feel we always want to throw when the context is not found. shouldn't we actually use a useContextOrThrow everytime and make a linter rule to avoid using useContext?